product-service/serverless.ts

```diff
@@ -24,6 +24,11 @@ const serverlessConfiguration: Serverless = {
         },
         environment: {
             AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+            PG_HOST: '${env:PG_HOST}',
+            PG_PORT: '${env:PG_PORT}',
+            PG_DATABASE: '${env:PG_DATABASE}',
```

Comment:
`env:PG_DATABASE` - you can use it directly in the place, see no reason to have it also in serverless.ts 🤔

Reply:
I believe that describing env vars in serverless.ts would configure values used in Lambda after deployment, wouldn't it?

Reply:
Looks like yes & no. You can easily use .env vars just in the handler and all magic will be resolved...